pkg/process/metrics.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package process
+
+import (
+	"context"
+	"flag"
+	"os"
+	"path/filepath"
+
+	"github.com/zeebo/admission/admproto"
+	"gopkg.in/spacemonkeygo/monkit.v2"
+	"gopkg.in/spacemonkeygo/monkit.v2/environment"
+	"storj.io/storj/pkg/telemetry"
+)
+
+var (
+	metricInterval = flag.Duration("metrics.interval", telemetry.DefaultInterval,
```

Comment:
I don't think library code should take flags

Reply:
same argument here - whether or not i agree library code in general should take flags, the goal of this specific library is to simplify boilerplate for process setup.